domain/src/main/java/edu/tamu/catalog/domain/model/HoldingsRecord.java

```diff
@@ -0,0 +1,41 @@
+package edu.tamu.catalog.domain.model;
+
+import java.util.Map;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.Setter;
+
+@AllArgsConstructor
```

Comment:
For non entities, we can use [@Data lombok](https://projectlombok.org/features/Data) annotation. Is there any reason we do not want to declare the properties final?